styles.css

```diff
@@ -0,0 +1,500 @@
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 10px;
+  text-align: center;
+  align-items: center;
+}
+
+header {
+  background-color: var(--green-gray);
+  grid-area: header;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.title-section {
+  text-align: center;
+  padding: 40px;
+  background-color: var(--dark-gray);
+}
+
+.title-flex-1 {
+  margin-top: -25px;
+  float: right;
+}
+
+.title-img {
+  width: 30px;
+  background-color: gray;
+  padding: 10px 20px;
+  border-radius: 10px;
+  margin-right: 10px;
+}
+
+footer ul {
+  display: flex;
+  flex-wrap: wrap;
+  width: 80%;
+  margin: 0 auto;
+  justify-content: space-evenly;
+}
+
+footer ul li {
+  margin: 0 15px;
+}
+
+.navlink li {
+  list-style: none;
+  color: white;
+  font-size: large;
+  font-weight: 600;
+}
+
+.navlinka {
+  padding: 5px 10px 7px 10px;
+}
+
+.navlinka:hover {
+  height: 30px;
+  background-color: var(--dark-gray);
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.navlinkmore {
+  background-color: var(--black);
+  padding: 5px;
+  border-radius: 5px;
+}
+
+.logoimage-style {
+  height: 100%;
+  width: 300px;
+  font-size: 30px;
+}
+
+.searchinput-style {
+  padding: 15px 55px;
+  border: 1px solid var(--gray);
+  border-radius: 12px;
+}
+
+.sarchicon-style {
+  position: relative;
+}
+
+.fa-search {
+  position: absolute;
+  left: 3px;
+  color: var(--gray);
+  top: 15px;
+}
+
+main {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.flexcontainer {
+  display: flex;
+  align-items: center;
+}
+
+.flexcontainer > * {
+  margin: 0 8px;
+}
+
+.flexcontainer-1 {
+  display: flex;
+}
+
+.class1 {
+  padding: 90px 20px;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+  position: relative;
+  height: 600px;
+}
+
+.div1 {
+  background-color: var(--black);
+}
+
+.div2 {
+  background-color: var(--silver);
+}
+
+.div3 {
+  background-color: var(--green-gray);
+}
+
+.div4 {
+  background-color: var(--blue-gray);
+}
+
+.imgmain {
+  transform: rotate(-15deg);
+  border-radius: 10px;
+  cursor: pointer;
+}
+
+.imgmain:hover {
+  transform: none;
+}
+
+section {
+  grid-area: section;
+}
+
+.latest-posts {
+  display: flex;
+  height: 550px;
+  cursor: pointer;
+  margin-top: 10px;
+}
+
+.flex-box-1,
+.flex-box-2,
+.flex-box-3,
+.flex-box-5,
+.flex-box-6,
+.flex-box-7 {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 20px;
+}
+
+body span {
+  color: transparent;
+  background-color: var(--gray);
+  line-height: 30px;
+}
+
+.flex-box-1 {
+  background-color: var(--dark-gray);
+}
+
+.flex-box-2 {
+  background-color: var(--l-gray);
+}
+
+.flex-box-3 {
+  background-color: var(--dark-gray);
+}
+
+.flex-box-5 {
+  background-color: var(--silver);
+}
+
+.flex-box-6 {
+  background-color: var(--blue-gray);
```

Comment:
## Grayscale

All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images if the students decide to add any ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/grayscale_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown))

- [ ] Remove all blues and green gray just keep gray with B/N colors

Reply:
It's done.